resources/prosody-plugins/mod_muc_allowners.lua

```diff
@@ -1,12 +1,99 @@
+local jid = require "util.jid";
```

Comment:
Does it still make sense to call this module "allowners"?

Reply:
Nope, we can copy it under a different name, but we should keep the old one as we will break stuff by renaming it.